### PR TITLE
Fix: Issue #30 lost WSConn readBytes count & UDP lost writePkgNum count

### PR DESCRIPTION
**Pull request by watermelo**
#30

### CODE REVIEW

File: transport/client_test.go [hunk 1]
```diff
@@ -122,7 +122,14 @@ func TestTCPClient(t *testing.T) {
 	ss.SetCompressType(CompressNone)
 	conn := ss.(*session).Connection.(*gettyTCPConn)
 	assert.True(t, conn.compress == CompressNone)
+	beforeWriteBytes := conn.writeBytes
```
Comment (AlexStocks) [inline transport/client_test.go]: pls read conn.writeBytes/conn.writePkgNum in atomic way in this test. thx.

Comment (watermelo) [reply transport/client_test.go]: ok~

Comment (chinaerserver) [reply transport/client_test.go]: I still have plazz  : where to use incReadPkgNum()  incWritePkgNum(),
about incReadPkgNum() it appear in connection.go websocket recv() method but incWritePkgNum() doesn't appear in connection.go websocket send() method.
In this pr ,add incWritePkgNum() in connection.go udp send() method.
I search all place of code,found that
incReadPkgNum() invoke in session.go addTask() method
incWritePkgNum() invoke in session.go WritePkg() WriteBytes() WriteBytesArray()
so what is the meaning of pkg , i'm confused

Comment (watermelo) [reply transport/client_test.go]: @chinaerserver sorry. It's my fault and fixed.
Pkg means application packet. `WritePkg` used to handle application packet. so `WritePkgNum` means number of packets processed. 

`send` and `recv` used to send binary data. 

`addTask()` method  used to dispach the Pkg to event handler. so `incReadPkgNum`  is here.

Comment (chinaerserver) [reply transport/client_test.go]: In my opinion ,pkg means user defined struct , encode by Write interface into byteArray .  Is my understanding correct ?
so incPkgReadNum() and incRkgWriteNum() function should be only used in session level , in connecttion level only used incReadBytes() and incWriteBytes()

Comment (chinaerserver) [reply transport/client_test.go]: @watermelo i have seen your newest commit , I can't agree with you any more!

Comment (watermelo) [reply transport/client_test.go]: Thanks~~

**Comment by AlexStocks** [comment]
LGTM

**Pull request merged**